demo/apiExamples.md

```diff
@@ -1,18 +1,26 @@
 <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../api.md) -->
 <!-- The below content is automatically added from ./../api.md -->
 
-# auro-themeswitcher
+#### <a name=""></a>``<a href="#auro-themeswitcher" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>

```

Comment:
Are these necessary? Didn't seem like it when I ran the localhost.

Reply:
I don't think so these were added when I ran a build.  I can revert that part of the commit.